doc/md_src_files/040_stakeholders-scope.md

```diff
@@ -23,12 +23,61 @@ Relatedly, C-level executives and public policy professionals often make, shape,
+We elucidate these ideas with the following examples.
 
-### Supplying Patches
+### Supplier Units of Work
+
+On the input side of the Supplier process, Suppliers typically receive reports of vulnerabilities in one or more versions of their product. 
+Part of the Supplier's task on initial report intake is to resolve the initial report into a set of products and versions that are affected by the reported vulnerability. 
+On the output side, Suppliers provide fix bundles for affected products.
+
+Our working assumption is that for SSVC purposes, the Supplier's unit of work is the tuple of (vulnerability, affected product). 
+This implies the need for Suppliers to be able to resolve whatever they receive to that level of granularity in order to make best use of SSVC.
+
+There are a variety of ways a Supplier might need to resolve the set of affected products. For example they might
+
+* recognize, on further investigation of the initial report, that additional versions of the product are affected
+* discover that other products are affected due to code sharing or even consistent programmer error across products
+* receive reports of vulnerabilities in third party libraries they include in one or more of their products
+* receive fix bundles for third party libraries used in one or more of their products (where a fix bundle might resolve multiple vulnerabilities or add new features)
+
+We note that SBOM seems well-placed to aid in that resolution process for the third party library scenarios.
+
+
+### Deployer Units of Work ###
+
+Deployers are usually in the position of receiving fix bundles from their Suppliers for products they have deployed. 
+They are then faced with the decision to deploy the fix bundle to a particular instance or not. 
+Whether or not they have the option of deploying only part of a fix bundle depends on whether the Supplier has engineered their release process to permit that degree of flexibility. 
+For example, if service packs are fix bundles, the Supplier might choose to release individually deployable fixes as well.
 
+The vulnerability management process for deployers has at its core the collation of data including
+* an inventory of deployed instances of product versions
+* a mapping of vulnerabilities to fix bundles
+* a mapping of fix bundles to product versions
+The first of these must be collected by the deployer, while the latter two most often originate from the product Supplier.
+
+In turn, deployers must resolve this information into specific actions in which a fix bundle is slated for deployment to replace or modify a particular instance of the product.
+The deployer tree in SSVC takes into account the mission and safety risks inherent to the category of systems to which those deployed instances belong.
+For this reason we recommend that the (fix bundle, product version instance) pairing constitutes the unit of work most appropriate for the SSVC.
```

Comment:
s/pairing/tuple
If you're going to introduce the jargon, stick with the same jargon. 

Also I don't think this is quite right. The pairing should match the output from the supplier, which may include mitigations, not just remediations (that is, fix bundles). And remediation is a bit broader than fix bundle, anyway. 
So while (fix bundle, instance) is the most common unit of work, we should say what the other possible units of work are.
IMO, SSVC works just as well for a deployer on (network config change, product version instance) as it does for a fix bundle. So I don't think this causes any downstream problems?

Reply:
I would swap out "fix bundles" for
 "mitigation or remediation".  
"update"
"software update"

In an early part of the "units of work" define _software update_  as  either a mitigation or remediation.

Reply:
and then you can refer to the _software update_ and cover both possibilities.

Reply:
fixed in e00074cf7b7523262685dca958bee1458aa7f2d3

I went with "remediation or mitigation" as the core phrase, adapting as appropriate in context